profiles/desc/cpu_flags_arm.desc

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License v2
+
+aes - Use the AES cryptography instruction set
+crc32 - Use the CRC32 instruction set
+edsp - Use the enhanced DSP instructions (ARMv*E and ARMv6+)
+iwmmxt - Use the iwMMXt instruction set
+neon - Use the NEON (ASIMD) instruction set
+sha1 - Use the SHA-1 cryptography instruction set
+sha2 - Use the SHA-2 cryptography instruction set
+thumb - Enable Thumb instruction set (ARMv*T and ARMv6+)
+thumb2 - Enable Thumb-2 instruction set (ARMv*T2 and ARMv7+)
+v4 - Use ARMv4 instructions
+v6 - Use ARMv6 instructions
+v7 - Use ARMv7 instructions
+v8 - Use ARMv8 instructions
+vfp - Use the VFP instruction set
+vfp-d16 - Indicate that the FPU has 16 64-bit VFP registers
+vfp-d32 - Indicate that the FPU has 32 64-bit VFP registers
+vfpv3 - Use the VFP version 3 instruction set
```

Comment:
Based on https://www.arm.com/products/processors/technologies/vector-floating-point.php
I'd merge vfp-d16 with vfpv3

VFPv3 is an optional extension to the ARM, Thumb® and ThumbEE instruction sets in the ARMv7-A and ARMv7-R profiles. VFPv3 implementation is with either thirty-two or sixteen double word registers. The terms VFPv3-D32 and VFPv3-D16 distinguish between these two implementation options.